### DISCUSSION TITLE
Is there is a way to enable citation completion in source mode Citation completion in RStudio source mode?

### DISCUSSION THREAD

**Discussion by hhilbig** [quarto-dev/quarto-cli#4566] [Q&A] [answered]
Hey, I just started using Quarto, and I was wondering if there is a way to enable citation completion in source mode (ie typing "@" and then getting suggested citations). This feature works for me in visual mode (I use a bibtex bibliography file), but I prefer editing documents in source mode. If this feature does not exist, it would be great if it could be added.

**Comment by mcanouil** [comment]
You might want to look for this and/or ask for this in <https://github.com/rstudio/rstudio>.
This feature is an IDE feature and not Quarto CLI feature.

**Reply by fgeeri** [reply, marked as answer]
This seems to be a popular feature request, and there is an issue for it: rstudio/rstudio#7876
